doc/man3/ASYNC_start_job.pod

```diff
@@ -112,7 +112,14 @@ with them. Applications can wait for the file descriptor to be ready for "read"
 using a system function call such as select or poll (being ready for "read"
 indicates that the job should be resumed). If no file descriptor is made
 available then an application will have to periodically "poll" the job by
-attempting to restart it to see if it is ready to continue.
+attempting to restart it to see if it is ready to continue. From openssl 1.1.2,
```

Comment:
The next version of OpenSSL will be 3.0.0

Reply:
changed